packages/storybook/src/nimble/theme-provider/tokens.ts

```diff
@@ -0,0 +1,185 @@
+
+import {
+    bodyFont,
+    bodyFontColor,
+    groupHeaderFont,
+    groupHeaderFontColor,
+    groupHeaderTextTransform
+} from '../../../../nimble-components/src/theme-provider/design-tokens';
+import { createUserSelectedThemeStory } from '../../utilities/storybook';
+
+export type TokenName = keyof typeof tokens;
+
+export interface TokenArgs {
+    propertyFormat: PropertyFormat;
+}
+
+const computedCSSValueFromTokenName = (tokenName: string): string => {
+    return getComputedStyle(document.documentElement).getPropertyValue(
+        cssPropertyFromTokenName(tokenName)
+    );
+};
+
+const colorTemplate = html<TokenName>`
+    <div
+        title="${x => computedCSSValueFromTokenName(tokens[x])}"
+        style="
+        display: inline-block;
+        height: 24px;
+        width: 24px;
+        border: 1px solid black;
+        background-color: var(${x => cssPropertyFromTokenName(tokens[x])});
+    "
+    ></div>
+`;
+
+const rgbColorTemplate = html<TokenName>`
+    <div
+        title="${x => computedCSSValueFromTokenName(tokens[x])}"
+        style="
+        display: inline-block;
+        height: 24px;
+        width: 24px;
+        border: 1px solid black;
+        background-color: rgba(var(${x => cssPropertyFromTokenName(tokens[x])}), 1.0);
+    "
+    ></div>
+`;
+
+const stringValueTemplate = html<TokenName>`
+    <div style="display: inline-block;">
+        ${x => computedCSSValueFromTokenName(tokens[x])}
+    </div>
+`;
+
+const fontTemplate = html<TokenName>`
+    <div
+        style="
+        display: inline-block;
+        font: var(${x => cssPropertyFromTokenName(tokens[x])});
+    "
+    >
+        Nimble
+    </div>
+`;
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const tokenTemplates: {
+    readonly [key in TokenSuffix]: ViewTemplate<TokenName>;
+} = {
+    Color: colorTemplate,
+    RgbPartialColor: rgbColorTemplate,
+    DisabledFontColor: colorTemplate,
+    FontColor: colorTemplate,
+    FontLineHeight: stringValueTemplate,
+    FontWeight: stringValueTemplate,
+    FontSize: stringValueTemplate,
+    TextTransform: stringValueTemplate,
+    FontFamily: stringValueTemplate,
+    BoxShadow: stringValueTemplate,
+    MaxHeight: stringValueTemplate,
+    MinWidth: stringValueTemplate,
+    Font: fontTemplate,
+    Size: stringValueTemplate,
+    Width: stringValueTemplate,
+    Height: stringValueTemplate,
+    Delay: stringValueTemplate,
+    Padding: stringValueTemplate
+};
+/* eslint-enable @typescript-eslint/naming-convention */
+
+const templateForTokenName = (
+    tokenName: TokenName
+): ViewTemplate<TokenName> => {
+    const suffix = suffixFromTokenName(tokenName);
+    if (suffix === undefined) {
+        throw new Error(`Cannot identify suffix for token: ${tokenName}`);
+    }
+    const template = tokenTemplates[suffix];
+    return template;
+};
+
+export function getTokensStory(tokenNames: TokenName[]): StoryObj<TokenArgs> {
```

Comment:
storybook has a well established pattern for parameterizing stories, can we use that instead of making a new pattern?